packages/extension-browser/src/devtools/panel/views/Header.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import { getMessage } from '../utils/i18n';
+
+import Button from './Button';
+
+import * as styles from './Header.css';
+
+type Props = {
+    actionDisabled?: boolean;
+    actionName: string;
+};
+
+const Header = ({ actionDisabled, actionName }: Props) => {
+    return (
+        <header className={styles.root}>
+            <div className={styles.actions}>
+                <Button type="submit" className={styles.button} primary={true} disabled={actionDisabled}>
```

Comment:
So I guess we have to use `Button` because that's how it's defined in `packages/extension-browser/src/devtools/panel/views/Button.tsx`? I'm not a fan of relying in capitalization for differences. Is this a common practice in react?

Reply:
We can call it whatever we want so long as it starts with a capital letter (which is what React requires). I'm fine renaming this to something like `AppButton` if we want to have a clearer visual difference compared to an HTML `button`.

Reply:
Also as far as the filename is concerned we can absolutely use lowercase or kebab-case if we prefer to align with the rest of the webhint codebase. Most React projects follow the convention of PascalCase filenames, but it is not required.

Reply:
I went ahead and changed everything to kebab-case to align better with the rest of the `hint` project.